s3/src/main/scala/akka/stream/alpakka/s3/javadsl/S3.scala

```diff
@@ -483,6 +483,27 @@ object S3 {
       .listBucket(bucket, prefix.asScala, s3Headers)
       .asJava
 
+  /**
+   * Will return a source of object metadata and common prefixes for a given bucket with optional prefix and delimiter using version 2 of the List Bucket API.
+   * This will automatically page through all keys with the given parameters.
+   *
+   * The `alpakka.s3.list-bucket-api-version` can be set to 1 to use the older API version 1
+   *
+   * @see https://docs.aws.amazon.com/AmazonS3/latest/API/v2-RESTBucketGET.html  (version 1 API)
```

Comment:
I assume this is actually the v2 API.  Can you update this reference and the one in `listBucket` javadoc?

Reply:
Those links seem to be outdated and redirect to https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html (and https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjects.html accordingly). I'll update the links and fix the version numbers

Reply:
Please do, see eg. https://github.com/akka/alpakka/pull/2024

Reply:
done. i've updated the other api links too